frontend/src/pages/account/Settings.vue

```diff
@@ -19,7 +19,10 @@
             </div>
         </template>
         <template v-else>
-            <ff-button @click="startEdit">Edit</ff-button>
+            <div class="flex space-x-4">
```

Comment:
The way this is presented, it feels like a 'cancel' button next to 'edit'.

I think it should appear more consistent with the 'Delete Project' button under Project/Instance settings:

<img width="676" alt="image" src="https://user-images.githubusercontent.com/51083/224720917-7dfc9579-359f-489b-8b4c-d7b02e997974.png">

The text can also say something about needing to delete/transfer team ownership as well. We shouldn't let the user try to delete their account if we know it's going to fail.

Reply:
updated in [7c26d2c](https://github.com/flowforge/flowforge/pull/1787/commits/7c26d2c2384c071a5586e301c08e35043bac3e07)

![image](https://user-images.githubusercontent.com/44235289/224746865-3ac1801e-2f2a-476b-87ac-74855b8c6e68.png)